app/assets/stylesheets/base/variables.scss

```diff
@@ -4,17 +4,26 @@
 Typically, if a color is used more than once or twice,
 it should be added here as a variable. Changing the value
 of these variables will affect the colors sitewide. */
-$font-stack-body: "proxima-nova", verdana, sans-serif;
-$font-stack-heading: "futura-pt", verdana, sans-serif;
-$cu-red: #98002e;
-$cu-red-rgb-transparent: rgba(168,36,57,0.95);
-// $cu-red2: #830c28; // Old red replaced by $cu-red to keep colors standard
-// $cu-red3: #98002E; // Old red replaced by $cu-red to keep colors standard
+$futura-regular: "futura-pt", verdana, sans-serif;
```

Comment:
Not sure why I didn't catch this before, but why is "Verdana" still in here? In all of our notes, we removed Verdana (see [the Trello card](https://trello.com/c/RYK2nRoa) for this story) so it's no longer in the font stacks. Should just be `arial, sans-serif` as the fallback fonts.